### PR TITLE
Ecf rails4.2.0

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

File: app/controllers/social_networking/likes_controller.rb [hunk 1]
```diff
@@ -18,9 +18,8 @@ def create
             .find(@like.item.item_id).participant_id
           notify Participant.find(like_item_participant_id)
         else
-          notify Participant.find(
-                   @like.item_type.constantize
-                   .find(@like.item_id).participant_id)
+          notify Participant.find(@like.item_type.constantize
```
Comment (gwensmu) [inline app/controllers/social_networking/likes_controller.rb]: my guess is that all these files are changed to satisfy rubocop, yes?

Comment (ericcf) [inline app/controllers/social_networking/likes_controller.rb]: indeed... :cop:

File: spec/controllers/social_networking/nudges_controller_spec.rb [hunk 2]
```diff
@@ -31,12 +31,15 @@ module SocialNetworking
         end
 
         before do
+          @routes = Engine.routes
```
Comment (gwensmu) [inline spec/controllers/social_networking/nudges_controller_spec.rb]: are these changes related to the ruby version update?

Comment (ericcf) [inline spec/controllers/social_networking/nudges_controller_spec.rb]: Related to the Rails update, yes.

**Comment by gwensmu** [comment]
looks legit ⛵
